src/lib/openjp2/tcd.c

```diff
@@ -1246,9 +1246,11 @@ static OPJ_BOOL opj_tcd_code_block_enc_allocate_data(opj_tcd_cblk_enc_t *
     /* and +7 for https://github.com/uclouvain/openjpeg/issues/1283 (-M 3) */
     /* and +26 for https://github.com/uclouvain/openjpeg/issues/1283 (-M 7) */
     /* and +28 for https://github.com/uclouvain/openjpeg/issues/1283 (-M 44) */
+    /* and +33 for https://github.com/uclouvain/openjpeg/issues/1283 (-M 4) */
+    /* and +63 for https://github.com/uclouvain/openjpeg/issues/1283 (-M 4 -IMF 2K) */
     /* TODO: is there a theoretical upper-bound for the compressed code */
     /* block size ? */
-    l_data_size = 28 + (OPJ_UINT32)((p_code_block->x1 - p_code_block->x0) *
+    l_data_size = 63 + (OPJ_UINT32)((p_code_block->x1 - p_code_block->x0) *
```

Comment:
can you add an extra indentation space to the next line ?

Reply:
Which line?

Reply:
line 1254. See https://travis-ci.org/github/uclouvain/openjpeg/jobs/745821635

Reply:
Fixed, thanks!

Reply:
I close it by accident. I submit the new one.